build_tools/buildkite/cmake/android/arm64-v8a/benchmark2.yml

```diff
@@ -43,7 +43,7 @@ steps:
       - "trace-captures-pixel-4-${BUILDKITE_BUILD_NUMBER}.tgz"
     timeout_in_minutes: "60"
 
-  - label: "Benchmark on Galaxy S20 (exynos-990, mali-g77)"
+  - label: "Benchmark on Pixel 6 Pro (google-tensor, mali-g78)"
```

Comment:
Can we add the Pixel 6 benchmark before removing the Samsung one?

Reply:
Done.